src/sentry/static/sentry/app/views/settings/organizationMembers/components/membersFilter.tsx

```diff
@@ -29,7 +29,9 @@ type Filters = {
 };
 
 const getBoolean = (list: string[]) =>
-  Array.isArray(list) ? list && list.map(v => v.toLowerCase()).includes('true') : null;
+  Array.isArray(list) && list.length
+    ? list && list.map(v => v.toLowerCase()).includes('true')
+    : null;
```

Comment:
@EvanPurkhiser I updated this fyi. Let me know if this is fine. I updated `getTagValues()` in tokenized search to always return `string[]`.